snooty/test_devhub.py

```diff
@@ -1,130 +0,0 @@
-from pathlib import Path
-from typing import Any, Dict, List, cast
-
-import pytest
-
-from .n import FileId, SerializableType
-from .parser import Project
-from .test_project import Backend
-from .types import BuildIdentifierSet
-from .util_test import ast_to_testing_string, check_ast_testing_string
-
-
-@pytest.fixture
-def backend() -> Backend:
-    backend = Backend()
-    build_identifiers: BuildIdentifierSet = {}
-    with Project(Path("test_data/test_devhub"), backend, build_identifiers) as project:
```

Comment:
Curious if we should delete the actual test data as well for `test_devhub`? I don't see any other references to it

Reply:
Oop! Yes, probably, forgot about test_data 😅